src/extensions/common/NiceTable.tsx

```diff
@@ -109,6 +142,51 @@ const NiceTable: FunctionComponent<Props> = ({
     );
 };
 
+const DeleteRowButton: FunctionComponent<{title: string, rowKey: string, onDeleteRow?: (key: string) => void}> = ({ title, rowKey, onDeleteRow }) => {
+    const handleClick = useCallback(() => {
+        onDeleteRow && onDeleteRow(rowKey)
+    }, [onDeleteRow, rowKey])
+    return (
+        <IconButton
+            title={title}
+            onClick={handleClick}
+        ><Delete /></IconButton>
+    )
+}
+
+const ConfirmDeleteRowButton: FunctionComponent<{title: string, rowKey: string, onConfirmDeleteRow?: (key: string, confirmed: boolean) => void}> = ({ title, rowKey, onConfirmDeleteRow }) => {
+    const handleClick = useCallback(() => {
+        onConfirmDeleteRow && onConfirmDeleteRow(rowKey, true)
+    }, [onConfirmDeleteRow, rowKey])
+    const handleCancel = useCallback(() => {
+        onConfirmDeleteRow && onConfirmDeleteRow(rowKey, false)
+    }, [onConfirmDeleteRow, rowKey])
+    return (
+        <span>
+            Confirm delete?
+            <IconButton
+                title={title}
+                onClick={handleClick}
+            ><Delete /></IconButton>
+            <IconButton
+                title={"Cancel"}
+                onClick={handleCancel}
+            >🗙</IconButton>
```

Comment:
This is a cancel emoji. For some reason, it's not displaying correctly with my installed font set, either on github or in vscode or (rather more importantly) through the web ui. To ensure consistency across users' font sets, we should probably encode it with a `FontAwesome`/`Material`/etc embedded font icon as we do elsewhere.

Reply:
I have replaced it by `&#10006;` which should do the trick.

Reply:
Confirming it looks good now.